spec/collator.html

```diff
@@ -49,46 +49,39 @@
-        11. Else
+        1. Else
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _u_ be ? GetOption(_options_, *"usage"*, *"string"*, «*"sort"*, *"search"*», *"sort"*).
+        1. Set _collator_.[[usage]] to _u_.
+        1. If _u_ is *"sort"*, then
+          1. Let _localeData_ be the value of *%Collator%*.[[sortLocaleData]].
+        1. Else,
           1. Let _localeData_ be the value of *%Collator%*.[[searchLocaleData]].
-        12. Let _opt_ be a new Record.
-        13. Let _matcher_ be GetOption(_options_, *"localeMatcher"*, *"string"*, «*"lookup"*, *"best fit"*», *"best fit"*).
-        14. ReturnIfAbrupt(_matcher_).
-        15. Set _opt_.[[localeMatcher]] to _matcher_.
-        16. For each row in <emu-xref href="#table-1">Table 1</emu-xref>, except the header row, do:
+        1. Let _opt_ be a new Record.
+        1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, «*"lookup"*, *"best fit"*», *"best fit"*).
+        1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. For each row in <emu-xref href="#table-1">Table 1</emu-xref>, except the header row, do:
           1. Let _key_ be the name given in the Key column of the row.
           1. Let _prop_ be the name given in the Property column of the row.
           1. Let _type_ be the string given in the Type column of the row.
           1. Let _list_ be a List containing the Strings given in the Values column of the row, or *undefined* if no strings are given.
-          1. Let _value_ be GetOption(_options_, _prop_, _type_, _list_, *undefined*).
-          1. ReturnIfAbrupt(_value_).
+          1. Let _value_ be ? GetOption(_options_, _prop_, _type_, _list_, *undefined*).
           1. If the string given in the Type column of the row is *"boolean"* and value is not *undefined*, then
-            1. Let _value_ be ToString(_value_).
-            1. ReturnIfAbrupt(_value_).
+            1. Let _value_ be ? ToString(_value_).
           1. Set _opt_.[[&lt;_key_&gt;]] to _value_.
-        17. Let _relevantExtensionKeys_ be the value of *%Collator%*.[[relevantExtensionKeys]].
-        18. Let _r_ be ResolveLocale(*%Collator%*.[[availableLocales]], _requestedLocales_, _opt_, _relevantExtensionKeys_, _localeData_).
-        19. Set _collator_.[[locale]] to the value of _r_.[[locale]].
-        20. Let _k_ be 0.
-        21. Let _lenValue_ be Get(_relevantExtensionKeys_, *"length"*).
-        22. Let _len_ be ToLength(_lenValue_).
-        23. Repeat while _k_ < _len_:
+        1. Let _relevantExtensionKeys_ be the value of *%Collator%*.[[relevantExtensionKeys]].
+        1. Let _r_ be ResolveLocale(*%Collator%*.[[availableLocales]], _requestedLocales_, _opt_, _relevantExtensionKeys_, _localeData_).
+        1. Set _collator_.[[locale]] to the value of _r_.[[locale]].
+        1. Let _k_ be 0.
+        1. Let _lenValue_ be Get(_relevantExtensionKeys_, `"length"`).
```

Comment:
I personally think *"something"* makes the most sense but the current convention is indeed `"something"`.